devel/usbmuxd/Portfile

```diff
@@ -29,3 +29,23 @@ configure.cmd       ./autogen.sh
 configure.args      --disable-silent-rules \
                     --without-preflight \
                     --without-systemd
+
+subport usbmuxd-devel {
+    github.setup    libimobiledevice usbmuxd 45dd28e6a1ce8bcd42cc7d547c223b05ffebff9b
+    version         20200507
+    revision        0
+
+    checksums       rmd160  d1e8b40c7ca6e97dedd0d4b1f26ba33f7d2e799a \
+                    sha256  b4fe9d00b1ac51a6627dedc96eae503ab12eb6695e3256f7ecb6195ce5744338 \
+                    size    59771
+
+    conflicts       usbmuxd
+
+    depends_lib-replace port:libplist port:libplist-devel
```

Comment:
Does this new devel port have to use the libplist devel port, or would either be OK ? If so the dep. should be expressed as a path style one, rather than an explicit port.

Reply:
All the devel ports I added/updated must use devel ports as dependencies, or build will fail due to missing symbols

Reply:
OK, fair enough.